fxgl-entity/src/main/kotlin/com/almasb/fxgl/entity/level/tiled/TilesetLoader.kt

```diff
@@ -89,7 +92,15 @@ class TilesetLoader(private val map: TiledMap, private val mapURL: URL) {
 
         for (i in 0 until layer.data.size) {
 
-            var gid = layer.data.get(i)
+            val tempGid = layer.data.get(i)
+
+            // Orientation is encoded in 3 most significant bits of '.tmx' UInt which is embedded in lower half of Long
+            val isHorizontallyFlipped = tempGid and 0x80000000L != 0L
```

Comment:
Could we use this format, for consistency?

https://github.com/AlmasB/FXGL/blob/6924aea33ec92a8f6eea9da3adfb637ba9208706/fxgl-entity/src/main/kotlin/com/almasb/fxgl/entity/level/tiled/TMXLevelLoader.kt#L392

So 

```
val isHorizontallyFlipped = tempGid and (1L shl 31) != 0L
```

Then we can refer to tmx documentation and it's clearer which bit is being checked.

Same comment for checks below.